hyperSpec/vignettes/fileio.Rmd

```diff
@@ -1232,22 +1232,22 @@ In order to gain speed, the spectra matrix should be preallocated after the firs
 
 
 A short examination of the files (`flu*.txt`{.r} in directory `txt.PerkinElmer`) reveals that the actual spectrum starts at line 55, after a line containing `\#DATA`.
-For now, no other information of the files is to be extracted.
+For now, no other information about the files is to be extracted.
 It is thus easier to skip the first 54 lines than searching for the line after `\#DATA`.
 
-A fully featured import function should support:
+A fully-featured import function should support:
 
 -  Reading multiple files by giving a pattern
 -  hand further arguments to `scan()`{.r}.
    This comes handy in case the function is used later to import other data types.
--  Also skipping 54 lines would be a weird default, so we rather require it to be given
+-  Also, skipping 54 lines would be a weird default, so we instead require it to be given
 explicitly.
--  The same applies for the axis labels: they should default to reasonable settings for
-fluorescence spectra, but it should be possible to change them if needed.
+-  The same applies to the axis labels: they should default to reasonable settings for
+fluorescence spectra, but it should be possible to change them.
 -  The usual log entry arguments should be supplied.
 -  A sanity check should be implemented: stop with an error if a file does not have the same
 wavelength axis as the others.
--  If no file can be found, an empty `hyperSpec`{.r} object is a reasonable result: There is no need
+-  If no file can be found, an empty `hyperSpec`{.r} object is a consistent result: There is no need
```

Comment:
Shouldn't "There" start with lower-case letter?

Reply:
https://writingcenter.unc.edu/tips-and-tools/semi-colons-colons-and-dashes/#:~:text=Here's%20our%20suggestion%3A%20generally%2C%20the,to%20capitalize%20it%20or%20not.